cloudfoundry-operations/src/main/java/org/cloudfoundry/operations/applications/DefaultApplications.java

```diff
@@ -227,8 +233,14 @@ public Mono<Void> disableSsh(DisableApplicationSshRequest request) {
             .zip(this.cloudFoundryClient, this.spaceId)
             .flatMap(function((cloudFoundryClient, spaceId) -> Mono.zip(
                 Mono.just(cloudFoundryClient),
-                getApplicationIdWhere(cloudFoundryClient, request.getName(), spaceId, sshEnabled(true))
+                getApplicationIdV3(cloudFoundryClient, request.getName(), spaceId)
```

Comment:
Hi @dmikusa,
Thank you for reviewing the PR. 

I checked your comment. Yes, the code is there multiple times. I believe that even if create a method like Mono<Tuple2<CloudFoundryClient,String>> getCloudFoundryClientAndApplicationIdV3, it will have a similar readability experience.
 
Please let me know if you still want me to update the method.